src/components/product/OutOfStock/out-of-stock.module.scss

```
@@ -0,0 +1,53 @@
+@import "src/styles/scaffold";
+
+.fs-out-of-stock[data-fs-out-of-stock] {
```

```suggestion
.fs-out-of-stock {
```

We decided to use only the class here.